tests/unit/TodoList1.spec.ts

```diff
@@ -117,7 +117,7 @@ describe('TodoList - UI', () => {
       }
     ]
 
-    const wrapper = mount(TodoList, { store, vuetify, propsData: { inProcessFiling: 0 } })
+    const wrapper = mount(TodoList, { store, vuetify })
```

Comment:
This is just cleanup -- these changes were not required for this ticket.

I was looking through these tests and realized that `inProcessFiling` is not a prop, so I deleted it from the mount statements.